### PR TITLE
Rename 'cfg_match' to 'match_cfg' to avoid collision of std nightly macro with the same name.

### PR DESCRIPTION
**Pull request by jsantell**
Fixes #191.

### CODE REVIEW

File: src/utils/macros.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ macro_rules! use_prelude { () => (
 )}
 
 /// I really don't get the complexity of `cfg_if!`…
-macro_rules! cfg_match {
+macro_rules! cfg_matches {
```
Comment (danielhenrymantilla) [inline src/utils/macros.rs:10]: Sorry for the nit, but could you go with:

```suggestion
macro_rules! match_cfg {
```

instead? `matches` conveys too much of a boolean meaning to me 😄

**Review by danielhenrymantilla — COMMENTED** [review]
Thanks for the swift PR! I have a naming nit but other than that it LGTM ✅

**Comment by jsantell** [comment]
Thanks for the speedy review! Updated to `match_cfg!`

**Review by danielhenrymantilla — APPROVED** [review]

**Comment by jsantell** [comment]
Thank you!

**Comment by danielhenrymantilla** [comment]
[Released](https://github.com/getditto/safer_ffi/releases/tag/v0.1.4) in `0.1.4` 🙏

**Pull request merged**